packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -210,12 +210,11 @@ export class WorkspaceCommandContribution implements CommandContribution {
                         validate: name => this.validateFileName(name, parent, true)
```

Comment:
just async/await rewrite, not semantic changes, correct?

Reply:
Yes